app/views/docs/oauth-providers/github.md

```diff
@@ -280,17 +280,15 @@ void main() async {
     
     client
         .setEndpoint('https://cloud.appwrite.io/v1') // YOUR API Endpoint
-        .setProject('[PROJECT_ID]') // YOUR PROJECT ID
-    ;
+        .setProject('[PROJECT_ID]'); // YOUR PROJECT ID
     
-    // OAuth Login, for simplest implementation you can leave both success and
-    // failure link empty so that Appwrite handles everything.
-    final future = account.updateSession('current');
-    future.then(function (response) {
-        console.log(response); // Success
-    }, function (error) {
-        console.log(error); // Failure
-    }); 
+    
+    try {
```

Comment:
Remember to add the comments back in!

Reply:
This is the same comment that we have for OAuth Login sample. I will modify and add it back here 👍